src/main/java/io/dropwizard/kafka/KafkaAdminClientFactory.java

```diff
@@ -79,6 +82,11 @@
     @Valid
     @JsonProperty
     protected SecurityFactory security;
+    @JsonProperty
+    protected boolean createTopics;
+    @Valid
+    @JsonProperty
+    protected Set<KafkaTopicFactory> topics;
```

Comment:
Maybe default this to an empty set and add a `@NotNull`, just to avoid needing to do null checks. That way, you'll only need to check that the set is empty or not.